worldedit-core/src/main/java/com/sk89q/worldedit/function/mask/Mask.java

```diff
@@ -94,16 +77,6 @@ default Mask optimize() {
         return value == null ? this : value;
     }
 
-    default Mask and(Mask other) {
-        Mask value = and(other);
-        return value == null ? MaskIntersection.of(this, other) : value;
-    }
-
-    default Mask or(Mask other) {
-        Mask value = or(other);
-        return value == null ? MaskUnion.of(this, other) : value;
-    }
-
```

Comment:
This code wasn't that great and we should encourage developers to use Intersection and Union API's. If we want to provide an easier way for developers to create an Intersection and Union then a new issue should be made detailing what a good API needs.